util.c

```diff
@@ -89,6 +94,17 @@ warning(const char *fmt, ...)
 	}
 }
 
+
+extern void tcpdump_exit(int status)
+{
+
+#ifdef USE_LIBSMI
+    smiExit();
```

Comment:
Please use tabs - or 8 spaces - for indentation, for consistency with the rest of the code.